bbb-graphql-server/metadata/actions.yaml

```diff
@@ -215,6 +215,12 @@ actions:
       handler: '{{HASURA_BBB_GRAPHQL_ACTIONS_ADAPTER_URL}}'
     permissions:
       - role: bbb_client
+  - name: pluginDataAnalyticsSendObject
```

Comment:
`pluginLearningDashboardPushEntry`, what do you think?

Reply:
Since last conversation I thought it would make more sense to put `pluginLearningAnalyticsDashboardSendData`

Reply:
Agreed, I would include "generic": `pluginLearningAnalyticsDashboardSendGenericData`.